src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/Activities/CreatePayment.cs

```diff
@@ -21,9 +21,9 @@ public CreatePayment(ICommandDispatcher commandDispatcher, ILogger<CreatePayment
         [FunctionName(nameof(CreatePayment))]
         public async Task Create([ActivityTrigger] CreatePaymentInput input)
         {
-            _logger.LogInformation("Creating Payment for apprenticeship incentive id {apprenticeshipIncentiveId}, pending payment id {pendingPaymentId}, collection period {collectionPeriod}", input.ApprenticeshipIncentiveId, input.PendingPaymentId, input.CollectionPeriod);
+            _logger.LogDebug("Creating Payment for apprenticeship incentive id {apprenticeshipIncentiveId}, pending payment id {pendingPaymentId}, collection period {collectionPeriod}", input.ApprenticeshipIncentiveId, input.PendingPaymentId, input.CollectionPeriod);
             await _commandDispatcher.Send(new CreatePaymentCommand(input.ApprenticeshipIncentiveId, input.PendingPaymentId, input.CollectionPeriod.Year, input.CollectionPeriod.Period));
```

Comment:
Shouldn't the log be in a decorator of the logger be a decorator of the _commandDispatcher?